e2e/utils.e2e.ts

```diff
@@ -0,0 +1,79 @@
+import {ElementFinder, browser, by, element, ProtractorBy} from 'protractor';
+
+/**
+ * A set of utility functions for writing E2E tests.
+ */
+export class E2EUtils {
```

Comment:
Couple of things I want to rearrange here:
* I think we should just export / use the standalone functions without them being in a class (similar to `coerceBooleanProperty` or `applyCssTransform` in core). 
* I want to avoid having one big "utils" file and instead divide the functions into more tightly-scoped files. I was thinking we could have something `util/asserts.ts`, `util/geometry.ts`, `util/actions.ts`, and `util/query.ts`